modules/test/playwright/tests/osb-faro-web/segments.spec.ts

```diff
@@ -303,7 +305,8 @@ test(
 		});
 
 		await test.step('Go to Analytics Cloud and Switch the property', async () => {
-			await navigateToACSitesPageViaURL({
+			await navigateToACPageViaURL({
+				acPage: ACPage.sitePage,
```

Comment:
Hey @marcosapmf, I think there should be a navigation to the segments page and you could remove line 317:
`await navigateTo({page, pageName: 'Segments'});`

```suggestion
				acPage: ACPage.segmentPage,
```

Reply:
Same comment for all occurrences of `navigateToACPageViaURL` after this point